backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -5084,4 +5084,23 @@ databaseChangeLog:
                    (gen_random_uuid(), 'HIV', 'LP17126-1')
       rollback:
         - sql:
-            sql: DELETE FROM ${database.defaultSchemaName}.supported_disease WHERE name = 'HIV';
\ No newline at end of file
+            sql: DELETE FROM ${database.defaultSchemaName}.supported_disease WHERE name = 'HIV';
+  - changeSet:
+      id: add-equipment-uid-type-to-device-type-disease
+      author: bob@skylight.digital
+      comment: Add column to Device Type Disease for equipment UID type to be populated by the LIVD sync
+      changes:
+        - addColumn:
+            tableName: device_type_disease
+            columns:
+              - column:
+                  name: equipment_uid_type
+                  type: text
+                  remarks: Equipment UID from LIVD sync to be used in the universal pipeline
+      rollback:
+        - dropColumn:
+            tableName: device_type_disease
+            columns:
+              - column:
+                  name: equipment_uid_type
+                  remarks: Drop equipment UID type from LIVD sync to be used in the universal pipeline
```

Comment:
I don't think "addColumn" needs a rollback specified. In the [docs, it shows that it supports auto-rollback.](https://docs.liquibase.com/change-types/add-column.html)

Reply:
+1